index.bs

```diff
@@ -1605,7 +1605,7 @@ The <dfn constructor for="XRWebGLLayer">XRWebGLLayer(|session|, |context|, |laye
   1. Let |layer| be a new {{XRWebGLLayer}}
   1. If |session|'s [=ended=] value is <code>true</code>, throw an {{InvalidStateError}} and abort these steps.
   1. If |context| is lost, throw an {{InvalidStateError}} and abort these steps.
-  1. If |context|'s [=XR compatible=] boolean is false, throw an {{InvalidStateError}} and abort these steps.
+  1. If |context|'s [=XR compatible=] boolean is false and |session|'s [=XRSession/mode=] is not {{XRSessionMode/"inline"}}, throw an {{InvalidStateError}} and abort these steps.
```

Comment:
Let's change this to use the [=immersive session=] definition instead, which should be a bit more future-proof.

> If |session| is an [=immersive session=] and |context|'s [=XR compatible=] boolean is &lt;code&gt;false&lt;/code&gt;, throw an {{InvalidStateError}} and abort these steps.